src/Features/LanguageServer/ProtocolUnitTests/Ordering/LongRunningNonMutatingRequestHandler.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable disable
```

Comment:
Did you mean 
```suggestion
#nullable enable
```

Reply:
lmao what the hell. It was an auto-generated class. Kind of shocked it did this 😄

Reply:
Actually it totally makes sense (learned after trying to change). This is a test class where we aggressively return `null`'s 😄. The project itself is nullable so we have to disable here in order to allow our test classes to be light-weight.